src/database/reimbursement.ts

```diff
@@ -0,0 +1,89 @@
+import {app} from "../config";
+import {getFirestore, doc, setDoc, getDoc, getDocs, collection, query, where, orderBy} from "firebase/firestore";
+import {createUser, User} from "../auth/types";
+
+const db = getFirestore(app);
+
+export type DepartmentEnum = "IT" | "Events" | "Competition" | "Education" | "Industry" | "Project" | "Diversity" | "Publicity" | "Product";
```

Comment:
Easier if you define an actual enum and the can just access the property like Department.Publicity

Reply:
I kept it as a group of strings so it's easier to use those strings as selection options though